apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -51,6 +55,9 @@ export const askAITutor = createAsyncThunk(
   'aitutor/askAITutor',
   async (chatContext: ChatContext, thunkAPI) => {
     const state = thunkAPI.getState() as {aiTutor: AITutorState};
+    const instructionsState = thunkAPI.getState() as {
+      instructions: InstructionsState;
+    };
```

Comment:
@kakiha11 I think you should be able to combine these!

```
const { aiTutor, instructions } = thunkAPI.getState() as {
    aiTutor: AITutorState;
    instructions: InstructionsState;
  };
```

Reply:
@ebeastlake That exact change didn't work, but I made updates to only call `thunkAPI.getState()` one time. There is a larger diff now since I needed to update `state` to `aiTutorState` throughout the rest of this export chunk.

Reply:
Did you also update the reference to instructions below it when you tested?

Reply:
Yep! The variable name is now `levelInstructions` and has been updated where it's used.